arches/app/views/auth.py

```diff
@@ -81,7 +82,13 @@ def get(self, request):
                 },
             )
 
-    @method_decorator(ratelimit(key="post:username", rate=settings.RATE_LIMIT, block=False))
+    @method_decorator(
+        ratelimit(
+            key="post:username",
+            rate=("{}/{}".format(int(settings.RATE_LIMIT.split("/")[0]) * 2, settings.RATE_LIMIT.split("/")[1])),
+            block=False,
+        )
```

Comment:
django-ratelimit accepts `None` or python callables, so let's make this a little more robust and check it's a string before splitting it